spec/consumers/purl_updates_consumer_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe PurlUpdatesConsumer do
+  let(:message) { instance_double(Racecar::Message, value: message_value) }
+  # subject(:process) { described_class.nesw.process(message) }
```

Comment:
Should this commented out line be deleted?

Reply:
Sure, removed.